pkg/web/apps/apps_http_handler.go

```diff
@@ -26,8 +29,13 @@ func NewHTTPHandler(e *echo.Echo, s Service) *HTTPHandler {
 func (a *HTTPHandler) GetApps(c echo.Context) error {
 	apps, err := a.Service.GetApps(c)
 
+	// If no apps have been found, return a HTTP Status code of 204 with no response body
+	if err == models.ErrNotFound {
+		return c.NoContent(http.StatusNoContent)
```

Comment:
I know it's an echo function not something you made, but it seems counter-intuitive to to have a specific function for http status "No Content", yet you still have to provide the status code :thinking:

Reply:
Ah, so it is not for the status code. it is `Context#NoContent(code int) can be used to send empty body with status code.`